.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--
+Before opening an issue, please make sure you have checked the documentation for help/answers at http://frameworkbenchmarks.readthedocs.io/en/latest/
+
+If this issue is regarding documentation, please open that issue at our documentation repo at https://github.com/TechEmpower/TFB-Documentation
+
+If this is a feature request please check the project roadmaps first at https://github.com/TechEmpower/FrameworkBenchmarks/projects
+
+If this is a request for a new language/framework please check/add to the wiki at https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Suggested-Frameworks,-Languages-and-Features
```

Comment:
Just for consistency, there should be a comma between "language/framework" and "please"